projects/vehicles/worlds/village_center.wbt

```diff
@@ -4,7 +4,6 @@ WorldInfo {
   ERP 0.6
   basicTimeStep 10
   FPS 40
-  coordinateSystem "NUE"
   gpsReference 46.5156 6.56525 0
   lineScale 5
   contactProperties [
```

Comment:
One of the dunes is floating in the wrong position
![i](https://user-images.githubusercontent.com/44834743/141067661-61373bd1-6460-4f9e-bfc5-bb31afa89a13.png)

The sand is in the wrong direction
![j](https://user-images.githubusercontent.com/44834743/141068038-193808c7-50ff-4fd1-80b4-0d8cb2b22bda.png)